hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -54,6 +54,21 @@
       tags = ConfigTag.CLIENT)
   private int streamBufferSize = 4 * 1024 * 1024;
 
+  @Config(key = "datastream.max.buffer.size",
+      defaultValue = "4MB",
+      type = ConfigType.SIZE,
+      description = "The maximum size of the ByteBuffer "
```

Comment:
Why do we need this max buffer size config here? if the total size of data in the buffer list equals "datastream.buffer.flush.size", it should be good enough.

Reply:
@bshashikant Currently there is no limit on the size of a ByteBuffer a client can send during write , this property would solve that and also this helps in calculating the putblock boundary effectively (putblock would occur for every 8 byteBuffers).

Reply:
I think, this is a good config to have. Thanks for the clarification.